### ISSUE TITLE
Accessibility - Poor Color Contrast with More Information Icon

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#120]
By changing the background image on the main screen, there is not enough color contrast for the white More information icon on the tan-colored background image.

**Comment by jfc3-dol** [comment]
The "More Info" icon on the light tan part of the background image still does not have enough color contrast.

**Comment by Abychalil** [comment]
More Info icon color is updated to black, which is more visible now. Please check and verify. Attached screen shot for your reference.
![S T_MoreinfoIcon-iOS-ST-95](https://user-images.githubusercontent.com/49654920/73088163-b248e580-3ea1-11ea-8ca7-8023f75446a2.jpeg)

**Comment by jfc3-dol** [comment]
@Abychalil ,

Nice job fixing the color contrast issue with the More information button.

The new More information icon ( #121318 ) on the tan-colored background ( #987F5C ) has enough color contrast now. Its color contrast ratio is 4.9:1 and needed to be 4.5:1 or higher to pass.

I did notice that the light gray right-pointing arrow ( #C4C4C4 ) on the white background above the image only has a color contrast ratio of 1.7:1. So I'm going to add a new issue to that, so we have it for our next build to work on.

**Comment by Abychalil** [comment]
Ok. Thanks @jfc3-dol   
We will be working on the new issue (#123) soon

**Comment by jfc3-dol** [comment]
@Abychalil,

You're welcome.

I forgot to close the issue, which I will do now.
